api/setting.go

```diff
@@ -68,3 +70,9 @@ func (find *SettingFind) String() string {
 	}
 	return string(str)
 }
+
+type SettingAppFeishuValue struct {
```

Comment:
exported: exported type SettingAppFeishuValue should have comment or be unexported (revive)